tsl/src/compression/compression.c

```diff
@@ -2033,7 +2063,13 @@ decompress_batches_for_insert(ChunkInsertState *cis, Chunk *chunk, TupleTableSlo
 						  decompressor.compressed_datums,
 						  decompressor.compressed_is_nulls);
 
+#if PG14_GE
+		write_logical_replication_msg_decompression_start();
+#endif
```

Comment:
how about just simply putting 
```suggestion
		write_logical_replication_msg_decompression_start();
```
here

and put the `#if` directives inside `write_logical_replication_msg_decompression_start` ; so that `#if` -s are not polluting the code everywhere...

Reply:
Done